to_do_list/src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
if you dont use this file remove it.
unused files/ functions/ classes make the code look much more than it actually is and eventually make it harder to understand.